.craft.yml

```diff
@@ -0,0 +1,14 @@
+minVersion: 0.24.0
+changelogPolicy: auto
+preReleaseCommand: pwsh scripts/bump-version.ps1
+targets:
```

Comment:
The is the "UPM" following in another PR?

Reply:
> The current plan is to stick to single repo and publish the three packages (one for each EngineVersion) as zip files as GH release artifacts.

I see.

Reply:
no more UPM, see PR description - for now we're sticking to a single repo